src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -184,13 +184,13 @@
     <value>Unable to retrieve certificates because the thumbprint is not valid. Verify the thumbprint and retry. </value>
   </data>
   <data name="WriteRequestComplete" xml:space="preserve">
-    <value>Writing web request completed. (Number of bytes remaining: {0})</value>
+    <value>Web request completed. (Number of bytes processed: {0})</value>
```

Comment:
Is the original "remaining" correct?  Meaning that it should be "Number of bytes to process"?

Reply:
I think it was wrong all along. This string is the message that is sent last when the stream has completed.

https://github.com/PowerShell/PowerShell/blob/aea561f08df3cc1c6ce92135db47604868fade2c/src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs#L366

in 5.1:
```powershell
$Job = Start-Job -ScriptBlock { 
    Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -outfile $env:temp\nuget.exe 
} | Wait-Job
$progress = $Job.ChildJobs[0].Progress.ReadAll()
$progress[-2].StatusDescription
$progress[-1].StatusDescription
```

result:
```none
Writing request stream... (Number of bytes written: 5010552)
Writing web request completed. (Number of bytes remaining: 5010552)
```

Reply:
Ok, your change looks good then.  Thanks!